src/pepr/operator/crd/validators/clusterconfig-validator.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2024 Defense Unicorns
+ * SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+ */
+
+import { PeprValidateRequest } from "pepr";
+import { isBase64 } from "../../controllers/utils";
+import { ClusterConfig } from "../generated/clusterconfig-v1alpha1";
+
+export async function validateCfgUpdate(req: PeprValidateRequest<ClusterConfig>) {
+  try {
+    validateCfg(req.Raw);
+  } catch (e) {
+    return req.Deny(`Failed to validate UDSConfig update: ${e}`);
```

Comment:
This message reads funny to me? Imo we have "successfully validated" the UDS Config, but it failed the validation?
```suggestion
    return req.Deny(`ClusterConfig failed validation: ${e}`);
```

But potentially that is weird too, since it would end up being `ClusterConfig failed validation: ClusterConfig Validation: namespace or name is invalid`. May need to rework the messaging here?

Reply:
changed to "Validation Failed: ClusterConfig: ..."